src/app/beer_garden/api/http/handlers/v1/system.py

```diff
@@ -41,17 +41,14 @@ class SystemAPI(BaseHandler):
         tags:
           - Systems
         """
-        include_commands = (
-            self.get_query_argument("include_commands", default="").lower() != "false"
-        )
+        # include_commands = (
+        #     self.get_query_argument("include_commands", default="").lower() != "false"
+        # )
```

Comment:
I added a TODO for this. `include_commands` is deprecated, but we can't rip it out of the v1 API. It's commented because I need to make a way to pass `exclude_fields` to the individual `get_system` call.